functions/src/repository/response-git-webhook-repository.ts

```diff
@@ -17,6 +34,199 @@ export interface ResponseGitWebhookRepositoryInput {
 export const onResponseGitWebhookRepository: HttpsFunction = https.onRequest((req: https.Request, res: Response) => {
   return cors(req, res, () => {
     Logger.info(`${req.protocol}://${req.hostname} ; onResponseGitWebhookRepository: success!`);
-    res.status(200).send();
+
+    const inputData: any = req.body;
+    let result: Promise<any>;
+
+    Logger.info(req.body);
+    Logger.info(Object.keys(inputData));
```

Comment:
`req.body` and `inputData` are the same. If we need both loggers, we should be consistent in the variable we use to reference the same data. I guess `inputData` makes more sense at this stage in the code